shopelectro/views/catalog.py

```diff
@@ -33,6 +32,18 @@ def get_paginated_page_or_404(objects, per_page, page_number):
         raise http.Http404('Page does not exist')
 
 
+class CategorySortingOption:
+    def __init__(self, index=0):
```

Comment:
>index

seems `number` may be more relevant, because "option number" sounds better then "option index". Decide yourself

Reply:
thought about it. Indexing - that's pretty what we are doing with sorting options.
Number is some semantical thing. But price doesn't have index/number value `0` buy it's semantic